plugins/services/src/js/components/modals/ServiceModals.js

```diff
@@ -49,8 +62,15 @@ class ServiceModals extends React.Component {
     let key = ActionKeys.SERVICE_DELETE;
 
     if (isGroup) {
-      deleteItem = force =>
-        this.props.actions.deleteGroup(service.getId(), force);
+      const groupFrameworks = "getFrameworks" in service
```

Comment:
Why do we need to check if `service` has a `getFrameworks` method? Please adjust the implication to make so that checks like this one are not needed.

Reply:
Removed, that was added just to be sure but yes the isGroup check already makes sure that is there.